modules/order/src/Entity/LineItem.php

```diff
@@ -158,11 +159,59 @@ public function setData($data) {
   /**
    * {@inheritdoc}
    */
+  public function addAdjustment(Adjustment $adjustment) {
+    $this->get('adjustments')->appendItem($adjustment);
+    $this->calculateTotal();
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setAdjustments(array $adjustments) {
+    $this->set('adjustments', $adjustments);
+    $this->calculateTotal();
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function removeAdjustment(Adjustment $adjustment) {
+    $this->get('adjustments')->removeAdjustment($adjustment);
+    $this->calculateTotal();
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdjustments() {
+    return $this->get('adjustments')->getAdjustments();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
   public function preSave(EntityStorageInterface $storage) {
     parent::preSave($storage);
+    $this->calculateTotal();
+  }
 
+  /**
+   * Causes the line item to recalculate its total.
```

Comment:
Causes the... is weird. Why not just "Recalculates the line item total price"? Also, if the getter is getTotalPrice, shouldn't this be recalculateTotalPrice()?